### PR TITLE
Joshua/futr 892 standardize examples structure in code samples and docs

### PR DESCRIPTION
**Pull request by Joshuafolorunsho**
* Update code samples.
* Update docs.
* Add environment variable setup step.
* Add section on how to run mastra.
image

### CODE REVIEW

File: docs/src/pages/guide/how-to/02-adding-tools.mdx [hunk 1]
```diff
@@ -96,6 +110,12 @@ async function main() {
 main();
 ```
 
+Run the code with:
+
+```bash copy
```
Comment (calcsam) [inline docs/src/pages/guide/how-to/02-adding-tools.mdx:115]: should default to npx tsx

File: docs/src/pages/guide/how-to/03-building-workflows.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ Workflows are defined in a dedicated directory. The directory path is configured
 
 Here's an simple example of a workflow that uses predefined tools to crawl a website, uses an LLM to structure the data, and then analyzes the sentiment of the content.
 
-```ts filename="src/workflows/index.ts" showLineNumbers copy
+```ts filename="src/mastra/blueprints/index.ts" showLineNumbers copy
```
Comment (calcsam) [inline docs/src/pages/guide/how-to/03-building-workflows.mdx:13]: we should be using /src/mastra/workflows not bleuprints

File: docs/src/pages/guide/how-to/01-creating-agents.mdx [hunk 3]
```diff
@@ -123,46 +147,56 @@ main();
...
 }
 
-console.log("\n\n✅ Recipe complete!");
+main();
+```
+
+Run the code with:
+
+```bash copy
+pnpx bun src/mastra/index.ts
```
Comment (calcsam) [inline docs/src/pages/guide/how-to/01-creating-agents.mdx:199]: should be `npx tsx`

Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/01-creating-agents.mdx:199]: `tsx` compiles to JS, and requires some configuration to get up and working.
`bun` comes shipped with a ts runner.

File: docs/src/pages/guide/how-to/03-building-workflows.mdx (hunk 2 above)
Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/03-building-workflows.mdx:13]: Got it!

File: docs/src/pages/guide/how-to/02-adding-tools.mdx (hunk 1 above)
Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/02-adding-tools.mdx:115]: tsx compiles to JS, and requires some configuration to get up and working.
bun comes shipped with a ts runner.

File: docs/src/pages/guide/how-to/01-creating-agents.mdx (hunk 3 above)
Comment (calcsam) [reply docs/src/pages/guide/how-to/01-creating-agents.mdx:199]: i'm okay with bun for now but `tsx` is way more standard. Let's ship this but talk more offline

File: docs/src/pages/guide/how-to/02-adding-tools.mdx (hunk 1 above)
Comment (calcsam) [reply docs/src/pages/guide/how-to/02-adding-tools.mdx:115]: i'm okay with bun for now but tsx is way more standard, eg 20x more npm downloads. Let's ship this but talk more offline

File: docs/src/pages/guide/how-to/01-creating-agents.mdx (hunk 3 above)
Comment (Joshuafolorunsho) [reply docs/src/pages/guide/how-to/01-creating-agents.mdx:199]: Got it.

**Pull request merged**